File: tests/expected/cast.cpp
#include <iostream>  // NOLINT(build/include_order)

#include "pycpp/runtime/builtins.h"  // NOLINT(build/include_order)
#include "pycpp/runtime/sys.h"       // NOLINT(build/include_order)

inline void main_func() {
  int16_t a = static_cast<int16_t>(1);
  auto b = a;
  std::cout << b;
  std::cout << std::endl;
  int64_t c = static_cast<int64_t>(1);
  auto d = c;
  std::cout << d;
  std::cout << std::endl;
  uint64_t e = static_cast<uint64_t>(1);
  auto f = e;
  std::cout << f;
  std::cout << std::endl;
}

int main(int argc, char** argv) {
  pycpp::sys::argv = std::vector<std::string>(argv, argv + argc);
  main_func();
}

File: tests/expected/cast.dart
// @dart=2.9
import 'package:sprintf/sprintf.dart';

main_func() {
  final int a = 1.toInt();
  final b = a;
  print(sprintf("%s", [b]));
  final int c = 1.toInt();
  final d = c;
  print(sprintf("%s", [d]));
  final int e = 1.toInt();
  final f = e;
  print(sprintf("%s", [f]));
}

main(List<String> argv) {
  main_func();
}
